README.md

```diff
@@ -0,0 +1,84 @@
+  - [x] Create a snap that runs in `devmode`
+  - [x] Convert the snap to `strict` confinement, or `classic` confinement if it qualifies
+  - [x] Register the snap in the store, **using the preferred upstream name**
+  - [ ] Add a screenshot to this `README.md`
+  - [x] Add install instructions to this `README.md`
+  - [ ] Update snap store metadata, icons and screenshots
+  - [x] Publish the confined snap in the Snap store beta channel
+  - [x] Update the install instructions in this `README.md`
+  - [ ] Post a call for testing in the Snapcraft Forum ["Snapcrafters" category](https://forum.snapcraft.io/c/snapcrafters/23) - [link]()
+  - [ ] Add the Snapcraft store account (snap-advocacy@canonical.com) as a collaborator to your snap in the [Dashboard](https://dashboard.snapcraft.io) and ask a [Snapcrafters admin](https://github.com/orgs/snapcrafters/people?query=%20role%3Aowner) to accept this request
+  - [ ] Fix all important issues found during testing
+  - [ ] Make a post in the Snapcraft Forum ["store-requests" category](https://forum.snapcraft.io/c/store-requests/19) asking for a transfer of the snap name from you to Snapcrafters - [link]()
+  - [ ] Ask a [Snapcrafters admin](https://github.com/orgs/snapcrafters/people?query=%20role%3Aowner) to fork your repo into github.com/snapcrafters, and configure the repo for automatic publishing into edge on commit
+  - [ ] Add the provided Snapcraft build badge to this `README.md`
+  - [ ] Publish the snap in the Snap store stable channel
+  - [ ] Update the install instructions in this `README.md`
+  - [ ] Post an announcement in the Snapcraft Forum ["Snapcrafters" category](https://forum.snapcraft.io/c/snapcrafters/23) - [link]()
+  - [ ] Ask the Snap Advocacy team to celebrate the snap - [link]()
+  - [ ] Submit a pull request or patch upstream that adds snap install documentation - [link]()
+  - [ ] Ask upstream if they are interested in maintaining the Snap. If they are:
+    - [ ] Fork the upstream project, add the snap build files and required assets/launchers to that repo and submit a pull request or patch - [link]()
+    - [ ] Add upstream contact information to the `README.md`
+    - If upstream accept the PR:
+      - [ ] Request upstream create a Snap store account
+      - [ ] Add upstream account as a collaborator on the snap
+      - [ ] Contact the Snap Advocacy team to request the snap be transferred to upstream
+
+If you have any questions, [post in the Snapcraft forum](https://forum.snapcraft.io).
+
+<!--
+## The Snapcrafters
+
+| [![Your Name](https://gravatar.com/avatar/bc0bced65e963eb5c3a16cab8b004431/?s=128)](https://github.com/yourname/) |
+| :---: |
+| [Your Name](https://github.com/yourname/) |
+--> 
+
+<!-- Uncomment and modify this when you have upstream contacts
+## Upstream
+
+| [![Upstream Name](https://gravatar.com/avatar/bc0bced65e963eb5c3a16cab8b004431?s=128)](https://github.com/upstreamname) |
+| :---: |
+| [Upstream Name](https://github.com/upstreamname) |
+-->
```

Comment:
Please update this to the same template as the Signal README if you can! :)

Reply:
oh, missed those. I'll get that amended